checks/cs50/2017/x/fifteen/checks.py

```diff
@@ -0,0 +1,152 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Fifteen(Checks):
+
+    @check()
+    def exists(self):
+        """fifteen.c exists."""
+        super().exists("fifteen.c")
+
+    @check("exists")
+    def compiles(self):
+        """fifteen.c compiles."""
+        # Remove sleeps from student code
+        self.spawn("sed -i='' '/#include <unistd.h>/a \\\n#define usleep(x)' fifteen.c").exit(0)
```

Comment:
@brianyu28 if a line like this fails, will student be confused, since we'll log that we expected an exit code of 0?